client/profile-wizard/style.scss

```diff
@@ -522,9 +522,8 @@
 	.woocommerce-business-extensions__benefits {
 		padding: 4px;
 		display: grid;
-		grid-template-columns: 1fr 1fr;
-		width: 600px;
-		max-width: 100%;
+		grid-template-columns: 1fr;
+		max-width: 592px;
```

Comment:
This is probably going to be an issue on smaller screens.  I usually use the `width: x; max-width: 100%` pattern to make sure it can shrink on smaller screen sizes.

<img width="378" alt="Screen Shot 2020-08-14 at 8 39 44 AM" src="https://user-images.githubusercontent.com/10561050/90250705-29959500-de45-11ea-822d-7917d4833505.png">